README.md

```diff
@@ -30,13 +30,40 @@ This blueprint assumes that you are familiar with GKE and
 To deploy this blueprint you need:
 
 - A [Google Cloud project](https://cloud.google.com/docs/overview#projects) with billing enabled.
-- An account with the [Project Owner role](https://cloud.google.com/iam/docs/understanding-roles#resource-manager-roles) on the project.
+- An account with either the [Project Owner role](#option-1-project-owner-role) (full access) or [Granular Access roles](#option-2-granular-access).
 - The `serviceusage.googleapis.com` must be enabled on the project. For more
   information about enabling APIs, see
   [Enabling and disabling services](https://cloud.google.com/service-usage/docs/enable-disable)
 - A Git repository to store the environment configuration.
 
-You create the infastructure using Terraform. The blueprint uses a local [Terraform backend](https://www.terraform.io/docs/language/settings/backends/configuration.html),
+### Service Account Roles & Permissions
+
+You can choose between **Project Owner** access (full access) or **Granular Access** for more fine-tuned permissions.
+
+#### Option 1: Project Owner role
+
+The service account will have full administrative access to the project.
+
+- `roles/owner`: Full administrative access to the project ([Project Owner role](https://cloud.google.com/iam/docs/understanding-roles#resource-manager-roles))
+
+---
+
+#### Option 2: Granular Access
+
+The service account will be assigned the following roles to limit access to required resources:
```

Comment:
Can you briefly describe why we need these roles?

Reply:
I mean: a few words about each role

Reply:
I added

Reply:
The contents are ok.

In order to make this more concise, can you please refactor this list as

- `role`: description.

instead of using a sub-list item for the description.

Thanks.

Reply:
Sure